src/rules.rs

```diff
@@ -8,6 +8,7 @@ use serde::{Deserialize, Serialize};
 pub use yara_sys::scan_flags::*;
 
 use crate::{errors::*, initialize::InitializationToken, internals, YrString};
+use crate::internals::{CallbackMsg, CallbackReturn};
```

Comment:
Can you maybe re-export this here? So peoples don't have to reference the `internals` modules.

Reply:
Fix